### ISSUE TITLE
Tabs element bug in RTL - multiple tabs not showing content

### ISSUE THREAD

**Issue by yaronlevi** [GeekyAnts/NativeBase#885]
RN@0.44.2
NativeBase@2.1.4
Android Nexus 5
System language is Hebrew (RTL)

When using multiple tabs like so:

```
<View style={styles.container}>
        <Tabs>
          <Tab heading="Tab1">
            <Text>sss</Text>
          </Tab>
          <Tab heading="Tab2">
            <Text>yyy</Text>
          </Tab>
          <Tab heading="Tab3">
            <Text>vvv</Text>
          </Tab>
        </Tabs>
      </View>
```

Only the first tab's content is shown.
When swiping to other tabs, they are empty.
I think react-native-tab-view has a similar issue.

**Comment by SupriyaKalghatgi** [comment]
NativeBase does not support RTL as of now.
We may consider this in future

**Comment by SupriyaKalghatgi** [comment]
#518

**Comment by saivyas** [comment]
had problem with native base tabs when I did forceRtl. Everything works but only native base doesn't show anything on initial load .

So I had a temporary solution

@ https://stackoverflow.com/questions/58736619/nativebase-tabs-issue-for-force-rtl-fix/58736621#58736621
